tests/test_import.py

```diff
@@ -1,4 +1,4 @@
-# This file is part of drp_tasks.
+# This file is part of pipe_tasks.
```

Comment:
You changed all the references in this file from drp_tasks (correct) to pipe_tasks (not correct).  (I assume an errant cut-and-paste).

Reply:
Fixed.